setup.py

```diff
@@ -46,20 +46,16 @@
 f.close()
 
 portrait_files = ["src/python/graphics/share/portraits.scr", ]
-cmip5_amip_json = glob.glob("data/CMIP_metrics_results/CMIP5/amip/*.json")
-cmip5_historical_json = glob.glob(
-    "data/CMIP_metrics_results/CMIP5/historical/*.json")
-cmip5_piControl_json = glob.glob(
-    "data/CMIP_metrics_results/CMIP5/piControl/*.json")
-demo_ACME_files = glob.glob("demo/ACME/*.py")
-demo_CSIRO_files = glob.glob("demo/CSIRO/*.py")
-demo_CSIRO_files += glob.glob("demo/CSIRO/*.png")
-demo_CSIRO_files += glob.glob("demo/CSIRO/*.sh")
-demo_GFDL_files = glob.glob("demo/GFDL/*.py")
-demo_GFDL_files += glob.glob("demo/GFDL/*.png")
-demo_NCAR_files = glob.glob("demo/NCAR/*.py")
-param_files = glob.glob("doc/parameter_files/*.py")
-
+demo_ACME_files = glob.glob("demo/external-setups/ACME/*.py")
```

Comment:
The json files in the demo directory will be used for various demo scripts


From: Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>>
Reply-To: PCMDI/pcmdi_metrics <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, July 19, 2016 at 1:52 PM
To: PCMDI/pcmdi_metrics <pcmdi_metrics@noreply.github.com<mailto:pcmdi_metrics@noreply.github.com>>
Cc: gleckler1 <pgleckler@llnl.gov<mailto:pgleckler@llnl.gov>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [PCMDI/pcmdi_metrics] 343 pjg simple demos (#386)


In setup.py<https://github.com/PCMDI/pcmdi_metrics/pull/386#discussion_r71420929>:

> @@ -46,20 +46,16 @@
>  f.close()
>
>  portrait_files = ["src/python/graphics/share/portraits.scr", ]
> -cmip5_amip_json = glob.glob("data/CMIP_metrics_results/CMIP5/amip/*.json")
> -cmip5_historical_json = glob.glob(
> -    "data/CMIP_metrics_results/CMIP5/historical/*.json")
> -cmip5_piControl_json = glob.glob(
> -    "data/CMIP_metrics_results/CMIP5/piControl/*.json")
> -demo_ACME_files = glob.glob("demo/ACME/*.py")
> -demo_CSIRO_files = glob.glob("demo/CSIRO/*.py")
> -demo_CSIRO_files += glob.glob("demo/CSIRO/*.png")
> -demo_CSIRO_files += glob.glob("demo/CSIRO/*.sh")
> -demo_GFDL_files = glob.glob("demo/GFDL/*.py")
> -demo_GFDL_files += glob.glob("demo/GFDL/*.png")
> -demo_NCAR_files = glob.glob("demo/NCAR/*.py")
> -param_files = glob.glob("doc/parameter_files/*.py")
> -
> +demo_ACME_files = glob.glob("demo/external-setups/ACME/*.py")


and why do we need to package these files? These are examples/docs we can link to these from some wiki. If yuor demo does not use it, then it shouldn't be packaged. No-one will look at it buried inside the python distribution.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/PCMDI/pcmdi_metrics/pull/386/files/628a8b02e66a9688e975e40f3b67531aa7d878f3#r71420929>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEV6rVIeVrjGcO_TuF__zF7o_sIbyA-Gks5qXTkIgaJpZM4JQJb4>.